.travis.yml

```diff
@@ -0,0 +1,21 @@
+sudo: false
+language: perl
+matrix:
+    fast_finish: true
+perl:
+   - '5.24'
+   - '5.22'
+   - '5.20'
+   - '5.18'
+   - '5.16'
+   - '5.14'
+before_install:
+   - git config --global user.name "TravisCI"
+   - git config --global user.email $HOSTNAME":not-for-mail@travis-ci.org"
+install:
+   - cpanm --quiet --notest Dist::Zilla Dist::Zilla::App::Command::cover App::cpm Devel::Cover::Report::Codecov
+   - cpm install -w 2 --mirror=http://cpan.cpantesters.org/ -g $(dzil authordeps --missing)
+   - cpm install -w 2 --mirror=http://cpan.cpantesters.org/ -g $(dzil listdeps --author --missing)
+
+script:
+   - dzil smoke --release --author && dzil cover -test -report codecov && dzil xtest
```

Comment:
There seems no `dist.ini` but only `Makefile.PL`.

Reply:
Oh, right: I'm used to rolling my own Makefile.PLs but copied this .travis.yml. Will see about consolidating, thanks.